fireant/database/data_blending.py

```diff
@@ -0,0 +1,29 @@
+from .type_engine import make_columns
+
+
+class DataBlender:
+    def __init__(self, primary_db, primary_type_engine, secondary_db, secondary_type_engine):
+        self.primary_db = primary_db
+        self.primary_type_engine = primary_type_engine
```

Comment:
i think we can add the type engines directly to the database model, so you can just use `self.primary_db.type_engine`. That would eliminate the possibility of passing mismatched args to this class